src/mpid/ch4/shm/src/Makefile.mk

```diff
@@ -12,6 +12,7 @@ noinst_HEADERS += src/mpid/ch4/shm/src/shm_impl.h    \
         src/mpid/ch4/shm/src/shm_am_fallback_recv.h  \
         src/mpid/ch4/shm/src/shm_am_fallback_rma.h   \
         src/mpid/ch4/shm/src/shm_am_fallback_send.h  \
+        src/mpid/ch4/shm/src/shm_am_fallback_part.h  \
```

Comment:
I think this commit, "ch4/shm: add partitioned comm in shm am " should be squashed with the `netmod` am fallback commit. They are doing the same thing and always need to be updated together anyway.

Reply:
done